client/components/login/login.test.js

```diff
@@ -110,6 +110,15 @@ describe("<Login /> interactions", () => {
     expect(wrapper.state("password")).toEqual("test password");
   });
 
+  it("should change state value when handleCheckBoxChange function is invoked", () => {
+    // phone_number should not be present if mobile_phone_verification is off
+    expect(wrapper.find("[name='phone_number']")).toEqual({});
+    wrapper
+      .find("#remember_me")
+      .simulate("change", { target: { checked: false, name: "remember_me" } });
+    expect(wrapper.state("remember_me")).toEqual(false);
+  });
+
```

Comment:
Good that you added one test, although looks not sufficient to prevent this feature from breaking.
Is there any measure you can include in this PR so that if someone changes the code and breaks this feature, the tests will find out?